File: examples/node/get.ts
import { ArkeselClient } from '@kalourmade/arkesel-sms';

const id = process.argv[2];
if (!id) {
  console.error('Usage: tsx get.ts <message_id>');
  process.exit(1);
}

const client = new ArkeselClient(process.env.ARKESEL_API_KEY!);
const details = await client.sms.get(id);

console.log(`${details.id} — ${details.message} (${details.status})`);

File: examples/python/get.py
import os
import sys

from kalourmade_arkesel import ArkeselClient

id = sys.argv[1] if len(sys.argv) > 1 else sys.exit("Usage: python get.py <message_id>")
client = ArkeselClient(os.environ["ARKESEL_API_KEY"])

details = client.sms.get(id)

print(f"{details.id} — {details.message} ({details.status.value})")
